lib/runtime/wasmer/exports.go

```diff
@@ -227,6 +228,58 @@ func (in *Instance) QueryCallFeeDetails(ext []byte) (*types.FeeDetails, error) {
 // TODO: use this in block verification process (#1873)
 func (in *Instance) CheckInherents() {}
 
+// GrandpaGenerateKeyOwnershipProof returns grandpa key ownership proof from runtime.
+func (in *Instance) GrandpaGenerateKeyOwnershipProof(authSetID uint64, authorityID ed25519.PublicKeyBytes) (
```

Comment:
This function is not test covered, can you add a test for it?

Reply:
Since the types are defined as the function arguments I don't know what I can pass in to make it fail, so I wrote a simple test for the happy path. However if there is a case I didn't think of please let me know and I can add it.

Reply:
You cannot fail the encode, and probably not the decode (unless the instance is mocked etc. which is, well for another day), but would it be possible to make Exec fail with bad authSetID+authorityID values? If not possible, the happy path only is fine.

Reply:
I do not believe I can, with invalid data (with respect to the runtime, not formatting) it seems to return an empty proof